arm-storage/2016-05-01/swagger/storage.json

```diff
@@ -0,0 +1,1301 @@
+          "type": "object",
+          "additionalProperties": {
+            "type": "string"
+          },
+          "description": "Gets or sets a list of key value pairs that describe the resource. These tags can be used for viewing and grouping this resource (across resource groups). A maximum of 15 tags can be provided for a resource. Each tag must have a key with a length no greater than 128 characters and a value with a length no greater than 256 characters."
+        },
+        "properties": {
+          "x-ms-client-flatten": true,
+          "$ref": "#/definitions/StorageAccountPropertiesCreateParameters"
+        }
+      },
+      "required": [
+        "sku",
+        "kind",
+        "location"
+      ],
+      "description": "The parameters used when creating a storage account."
+    },
+    "Endpoints": {
+      "properties": {
+        "blob": {
+          "readOnly": true,
+          "type": "string",
+          "description": "Gets the blob endpoint."
+        },
+        "queue": {
+          "readOnly": true,
+          "type": "string",
+          "description": "Gets the queue endpoint."
+        },
+        "table": {
+          "readOnly": true,
+          "type": "string",
+          "description": "Gets the table endpoint."
+        },
+        "file": {
+          "readOnly": true,
+          "type": "string",
+          "description": "Gets the file endpoint."
+        }
+      },
+      "description": "The URIs that are used to perform a retrieval of a public blob, queue, or table object."
+    },
+    "StorageAccountProperties": {
+      "properties": {
+        "provisioningState": {
+          "readOnly": true,
+          "type": "string",
+          "description": "Gets the status of the storage account at the time the operation was called.",
+          "enum": [
+            "Creating",
+            "ResolvingDNS",
+            "Succeeded"
+          ],
+          "x-ms-enum": {
+            "name": "ProvisioningState",
+            "modelAsString": false
+          }
+        },
+        "primaryEndpoints": {
+          "$ref": "#/definitions/Endpoints",
+          "readOnly": true,
+          "description": "Gets the URLs that are used to perform a retrieval of a public blob, queue, or table object. Note that Standard_ZRS and Premium_LRS accounts only return the blob endpoint."
+        },
+        "primaryLocation": {
+          "readOnly": true,
+          "type": "string",
+          "description": "Gets the location of the primary data center for the storage account."
+        },
+        "statusOfPrimary": {
+          "readOnly": true,
+          "type": "string",
+          "description": "Gets the status indicating whether the primary location of the storage account is available or unavailable.",
+          "enum": [
+            "Available",
+            "Unavailable"
+          ],
+          "x-ms-enum": {
+            "name": "AccountStatus",
+            "modelAsString": false
+          }
```

Comment:
@JasonYang-MSFT - We have built a validation tool that validates an example request body and the response against the schema defined in the swagger spec. The tool fails on an error. 
The service returns value for "statusOfPrimary" as "available" instead of PascalCase "Available". This causes our validation to fail. I think the service should normalize the value to PascalCase while sending the value back to the customer. It can be relaxed while accepting the value. This will ensure that applications do not break on deserialization.

Sample output of the validation tool
```
> Operation: StorageAccounts_GetProperties

                > x-ms-examples

                        > Scenario: GetProperties
                                > RESPONSE_BODY_VALIDATION_ERROR: Found errors in validating the response with statusCode "200" for example "GetProperties" in operation "StorageAccounts_GetProperties".
                                > InnerErrors:
                                  1. code: ENUM_MISMATCH
                                     message: No enum match for: available
                                     path: properties,statusOfPrimary
                                     description: Gets the status indicating whether the primary location of the storage account is available or unavailable.
```